features.md

```diff
@@ -16,19 +16,67 @@
    
 * ### Viewing Patients
 
-   1. Downloading Patients
+   Patients that you have created from the client or synced patients from the server are shown in the **Find Patients** section. From here you can select a patient to do numerous events, such as visiting the patient, capturing vitals and so forth.
 
-   2. Finding Patients
+   #### 1. Downloading Patients
 
-   3. Visits
+   ![Viewing and searching patients](assets/openmrs_android_viewing_patients_1.png)
 
-   4. Diagnosis
+   If this is your first time using the Android client, you are likely to see an empty list of patients in **Find Patients** and you'll want to download and synchronize patients from your server. To do this, click the `+` button from the action bar, then click the search button and enter a patient name. You'll be shown a list of (unsynchronized) patients matching the searched name.
 
-   5. Vitals
```

Comment:
Same as above

Reply:
[Changed to a small header](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R63), without numbering